fuel-vm/src/util.rs

```diff
@@ -280,7 +283,7 @@ pub mod test_helpers {
             let salt: Salt = self.rng.gen();
             let program: Witness = contract
                 .into_iter()
-                .flat_map(Opcode::to_bytes)
+                .flat_map(<[u8; 4]>::from)
```

Comment:
This seems less ergonomic to me than `to_bytes()`, as it forces me to think about how many bytes each instruction should take everytime I want to serialize some instructions. Maybe we should keep the `to_bytes()` as a trait fn for all instructions?